PyRDF/RDataFrame.py

```diff
@@ -43,9 +67,8 @@ def __init__(self, *args):
 
     def _get_vector_from_list(self, arg):
         """
-        Helper method to convert
-        a python list of strings
-        to a vector
+        converts a python list of strings
```

Comment:
Internal method, docstring should be in the form of comments.
Also uppercase `Converts`.